src/utils/config.ts

```diff
@@ -109,16 +135,9 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
     .map(_ => _.toLowerCase() === "true")
     .toUndefined(),
   SERVER_PORT: fromNullableE(-1)(process.env.SERVER_PORT)
-    .chain(_ => {
-      // tslint:disable-next-line: no-console
-      console.log(`Server port from .env is ${_}`);
-      return IntegerFromString.decode(_).mapLeft(() => -1);
-    })
-    .fold(() => {
-      // tslint:disable-next-line: no-console
-      console.log(`folding to 8080`);
-      return 8080;
-    }, identity),
+    .chain(_ => IntegerFromString.decode(_).mapLeft(() => -1))
+    .chain(_ => NonNegativeInteger.decode(_).mapLeft(() => -1))
```

Comment:
Sorry for my mistake, the `NonNegativeInteger` type decoding already happen into `IConfig` in [here](https://github.com/pagopa/hub-spid-login-ms/pull/11/files#diff-285dea82a685fcd275e82141070c164a79a6650d46573cfd6303381afc47b28bR95)
We can remove this 
```suggestion
```